src/index.js

```diff
@@ -22,18 +22,26 @@ export default class GeoBatch {
       clientId = null,
       privateKey = null,
       apiKey = null,
+      queriesPerSecond = 50,
       accessor = address => address
     } = {
       cacheFile: 'geocache.db',
       clientId: null,
       privateKey: null,
       apiKey: null,
+      queriesPerSecond: 50,
```

Comment:
Put the 50 in a config file — when it is used that often, it’s better to have it in just one place.